packages/cart/src/services/cart-module.ts

```diff
@@ -35,8 +33,8 @@ type InjectedDependencies = {
 
 export default class CartModuleService implements ICartModuleService {
   protected baseRepository_: DAL.RepositoryService
-  protected cartService_: CartService
-  protected addressService_: AddressService
+  protected cartService_: services.CartService
```

Comment:
comment: we should avoid importing from the services index since the cart-module is also part of it it could create circular imports issue. I think we should try to avoid them as most as possible. Cause what happen is that we export from services from an index file, all export gets evaluated one by one, the first one gets evaluated and then import from the index something that is not yet evaluated and we can end up trying to extends undefined. Here we probably wont have this issue, but i think it is more easy to try to avoid them the most possible wdyt?

Reply:
cc @carlos-r-l-rodrigues